bot.py

```diff
@@ -5,26 +5,69 @@
 
 client = commands.Bot(command_prefix = "$")
 
-conn = sqlite3.connect('users.db')
+conn = sqlite3.connect(':memory:')
 
 c = conn.cursor()
 
 c.execute("""CREATE TABLE IF NOT EXISTS users (

```

Comment:
DB Set up should be done as a separate script you run once when "installing" the bot

Reply:
If this is a small bot that's intended to be a single script that has an in-memory database, I could understand doing this `CREATE TABLE` inline.

Regardless, I think this should be abstracted out as a separate function.